src/components/mdx_template_view.tsx

```diff
@@ -99,7 +99,11 @@ const MdxTemplateView: FC<MdxTemplateViewProps> = ({
                 <b>{doc_name}</b>
             </center>
             <br />
-            <MdxWrapper Component={config.template} Project={project} />
+            <MdxWrapper
+                Component={config.template}
+                Project={project}
+                JobId={jobId}
```

Comment:
Why is jobId being added now, when it wasn't needed before?

Reply:
Manual J-related text was not displaying before. The jobId is required to enable this text based on the Manual J file present in the attachment for that particular installation.